infra/terraform/test-org/ci-triggers/locals.tf

```diff
@@ -50,6 +50,7 @@ locals {
     "pubsub-java-app"             = "terraform-pubsub-integration-java"
     "genai-rag"                   = "terraform-genai-rag"
     "cloud-client-api"            = "terraform-cloud-client-api"
+    "dataanalytics-eventdriven"   = "terraform-dataanalytics-eventdriven"
```

Comment:
The repo seems to be `click-to-deploy-solutions`, not `data-analytics-platform-event-driven`. https://github.com/GoogleCloudPlatform/click-to-deploy-solutions also does not seem to conform to go/jss-tf-authoring

Reply:
Hello @bharathkkb, the repo is this one (https://github.com/GoogleCloudPlatform/terraform-dataanalytics-eventdriven)

Reply:
Thanks for clarifying, the description threw me off!